misago/users/tests/test_anonymizeoldips.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import pytz
```

Comment:
Don't use `datetime` or `pytz`. Django has utils for creating and working with timezone aware `datetime` objects.